### PR TITLE
Finish user delete 2

### PR DESCRIPTION
**Pull request by hitshootnum1**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+  before_action :admin_user, only: :destroy
+  before_action :find_user, only: :destroy
+
+  def destroy
+    @user.destroy
+    flash[:info] = t "user.deleted"
+    redirect_to users_path
+  end
+
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: remove this line

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Admin::UsersController < ApplicationController
+
+  before_action :admin_user, only: :destroy
+  before_action :find_user, only: :destroy
+
+  def destroy
+    @user.destroy
+    flash[:info] = t "user.deleted"
```
Comment (vuducluan) [inline app/controllers/admin/users_controller.rb]: If not deleted?

File: config/routes.rb [hunk 3]
```diff
@@ -9,4 +9,9 @@
 
   resources :users
   resources :relationships, only: [:create, :destroy]
+
+  namespace :admin do
+    resources :users, only: :destroy
+  end
+
```
Comment (hellovietnam93) [inline config/routes.rb]: remove this line

File: app/controllers/admin/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class Admin::UsersController < ApplicationController
+
+  before_action :admin_user, only: :destroy
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: don't need only, it must be for all

File: app/controllers/admin/users_controller.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  before_action :find_user, only: :destroy
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: the same as above

File: app/controllers/admin/users_controller.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+class Admin::UsersController < ApplicationController
+
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: check login

File: app/controllers/admin/users_controller.rb [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  before_action :admin_user
+  before_action :find_user
+
+  def destroy
+    begin
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: if @user.destroy
..
else 
...

File: app/helpers/sessions_helper.rb [hunk 8]
```diff
@@ -41,4 +41,15 @@ def correct_user
...
     session[:forwarding_url] = request.url if request.get?
   end
+
+  def find_user
+    @user = User.find_by id: params[:id]
+  end
+
+  def admin_user
+    unless current_user.admin?
+      flash[:danger] = t "user.isnt_authorized"
```
Comment (hellovietnam93) [inline app/helpers/sessions_helper.rb]: flash should be "permisson not authorized"

**Pull request merged**